spec/function.dd

```diff
@@ -3782,18 +3782,20 @@ $(H2 $(LNAME2 function-attribute-inference, Function Attribute Inference))
 
 $(H2 $(LNAME2 pseudo-member, Uniform Function Call Syntax (UFCS)))
 
-        $(P A free function can be called with a syntax equivalent to that of
-        a member function of its first parameter type. The free function is
-        called a $(I UFCS function).
+        $(P A free function can be called like a member function when:)
+        $(UL
+        $(LI The member function does not exist for the expression on the left hand side)
+        $(LI The free function's first parameter type matches the left hand expression)
```

Comment:
It's not clear that this list is a conjunction, i.e. these points must both hold, not just one of them. It's also unclear what "left hand expression" refers to.

Reply:
Updated.